src/Microsoft.Data.Analysis.Interactive/Microsoft.Data.Analysis.Interactive.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<TargetFramework>netcoreapp3.1</TargetFramework>
+		<IsPackable>false</IsPackable>
+		<NoWarn>$(NoWarn);MSML_ParameterLocalVarName;SA1028</NoWarn>
+	</PropertyGroup>
+
+	<ItemGroup>
+		<PackageReference Include="Microsoft.CodeAnalysis.Analyzers" Version="3.0.0">
+			<PrivateAssets>all</PrivateAssets>
+			<IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
```

Comment:
This is odd, you can use `ExcludeAssets="compile"` instead if that's what you were trying to do.  In general we prefer attributes over child elements for metadata.

Reply:
Hmmm this was added in https://github.com/dotnet/corefxlab/pull/2931. I'm wondering if it was copied over from a sample in https://github.com/dotnet/interactive/tree/master/samples/extensions but I can't find it anymore. I'll test locally if we even need this anymore